scripts/lib/CIME/case_st_archive.py

```diff
@@ -313,19 +314,20 @@ def _archive_history_files(case, archive, archive_entry,
             if ninst_string:
                 if compname.find('mpas') == 0:
                     # Not correct, but MPAS' multi-instance name format is unknown.
-                    newsuffix = compname + '.*' + suffix
+                    newsuffix =                   compname + r'\d*'                   + r'\.' + suffix + r'\.'
```

Comment:
Some more spacing issues

Reply:
I did that to vertically line up the same elements in multiple lines,
to make it easier to see the differences between lines.  If this
goes against the CIME style standards, I'll remove the extra spaces.

Reply:
For the record, I find no issue with these spaces (they do increase readability IMHO).

Reply:
It's not my preference (personally I prefer a formatting tool to do my formatting thinking for me), but if others don't have a problem with it, then that's fine. Though the one on line 322 is missing a space.

Reply:
No issue, I'm fine with this PR